src/components/executive-education-2u/styles/_ExecutiveEducation2UErrorPage.scss

```diff
@@ -0,0 +1,61 @@
+$heading-red: #D23228;
+$heading-black: #002121;
+$error-chip-text-color: #002121;
+$error-chip-background-color: #FFFFFF;
+$error-chip-border-color: #F2F0EF;
+
+.executive-education-2u-error-illustration {
+    @extend .mb-4;
+    align-items: center;;
+}
+
+.executive-education-2u-error-heading {
+    @extend .mb-4;
+    align-items: center;
+    line-height: 60px;
+
+    .executive-education-2u-error-heading-red {
+        font-size: 3.75rem;
+        font-weight: 700;
+        color: $heading-red;
+        margin-right: 10px;
+    }
+
+    .executive-education-2u-error-heading-black {
+        font-size: 3.75rem;
+        font-weight: 700;
+        color: $heading-black;
+    }
+}
+
+.executive-education-2u-error-heading-details {
+    @extend .mb-4;
+    align-items: center;
+    font-family: 'Roboto Mono', monospace;
```

Comment:
I don't think we should define a custom `font-family`. Instead, I would recommend using the appropriate existing SCSS variable from Paragon for the desired font-family: `$font-family-monospace` ([source](https://github.com/edx/brand-edx.org/blob/8a578b89925ceb99b1194b9d86b52303b9ac0825/paragon/_variables.scss#L420)). That way, if the design system changes it's monospace font, the change will propagate here, too, for consistency.